File: src/main/java/training/katas/DeodorantEvaporator/deodorant_evaporator.js
function evaporator(content, evap_per_day, threshold) {
    let nthDay = 0;
    const absoluteThreshold = (content * threshold) / 100;
    while (content > absoluteThreshold) {
        nthDay += 1;
        content *= (100 - evap_per_day) / 100;
    }
    return nthDay;
}
module.exports = { evaporator };

File: src/main/java/training/katas/DeodorantEvaporator/deodorant_evaporator.go
package DeodorantEvaporator

func Evaporator(content float64, evapPerDay int, threshold int) int {
	var nthDay = 0
	var absoluteThreshold = content * float64(threshold) / 100
	for {
		nthDay += 1
		content *= float64((100.0 - float64(evapPerDay)) / 100.0)
		if content < absoluteThreshold {
			break
		}
	}
	return nthDay
}
